react/src/base/inputs/DatePicker.stories.js

```diff
@@ -69,56 +91,75 @@ export const invalidDatePicker = () => (
 invalidDatePicker.story = {
   name: 'Invalid',
   parameters: {
-    jest: ['SprkErrorContainer', 'SprkInputContainer'],
+    jest: [
+      'SprkErrorContainer',
+      'SprkInputContainer',
+      'SprkDatePicker',
+      'SprkLabel',
+      'SprkIcon',
+    ],
   },
 };
 
 export const disabledDatePicker = () => (
   <SprkInputContainer>
-    <SprkDatePickerInput
-      name="date"
-      placeholder="01/01/2019"
-      errorMessage="Incorrect date."
-      label="Date"
-      disabled
-    />
+    <div className="sprk-b-InputContainer__icon-container">
```

Comment:
Same as above:

Can this div be part of SprkInputContainer? not sure if it's worth it but I'd like to bring it up